src/components/ErrorBoundary.js

```diff
@@ -15,6 +18,27 @@ class ErrorBoundary extends Component {
 
 	static contextType = StoreContext
 
+	refreshFights = () => {
+		const {reportStore} = this.context
+		reportStore.refreshReport()
+	}
+
+	renderRefreshButton = (error) => {
+		if (error instanceof NotFoundError) {
```

Comment:
Should probably invert this conditional. Needs silly wrapping `(!(...))`, but better than all that indentation.